packages/@react-spectrum/inlinealert/test/InLineAlert.test.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {InlineAlert} from '../';
+import {render} from '@react-spectrum/test-utils';
+import React from 'react';
+
+describe('InlineAlert', function () {
+  it.each`
+    Name | Component      | props
+    ${'InlineAlert'} | ${InlineAlert} | ${{}}
+  `('$Name handles defaults', function ({Component, props}) {
```

Comment:
simplifying, we used to use this syntax everywhere for v2 or if there are multiple cases that can be handled by one test, but since we only have one example here, we can do it the less verbose way
```suggestion
  it('InLine Alert handles defaults', function () {
```